### DISCUSSION TITLE
dispositivi zigbee non disponibili

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1844] [HUB: Home Assistant]
Ciao, da qualche giorno(credo in concomitanza con l'aggiornamento alla versione 2022.8 tutti i miei dispositivi zigbee integrati con Z2M non sono più disponibili. Dalla dashboard di zqm funzionano perfettamente. Avete dei suggerimenti?
versione 2022.8, HA core installato via docker su raspberry.
Grazie.

![image](https://user-images.githubusercontent.com/60674280/185260476-3e418703-12e4-4520-8265-1ab2af2d1e80.png)

**Comment by snakuzzo** [comment]
Verifica che tra le entità non siano presenti entità duplicate

**Reply by calcioscacchi** [reply]
Negativo.  Solo una è duplicata,  ma è roba vecchia

**Reply by snakuzzo** [reply]
Dovresti controllare i topic mqtt...ma dovresti sapere da solo come fare. Impossibile da qui dirti cosa fare di preciso

**Reply by calcioscacchi** [reply]
Per vedere se ho capito... Dici di controllare le discussioni riguardo mqtt per vedere se atri hanno segnalato il problema?

**Reply by snakuzzo** [reply]
No ..devi metterti ad analizzare cosa passa sulla coda del broker mqtt...controllando i topic di autodiscovery.
Ma non posso dirti io come fare...mettiti e studia come funziona mqtt

**Reply by calcioscacchi** [reply]
li ho controllati e sembravano regolari. Ho controllato la dashboard z2m e i dispositivi erano in stato disabilitato. Spero di aver risolto come descritto sotto.

**Comment by calcioscacchi** [comment]
ciao, ieri sera ho scovato questo articolo sul sito di zigbee2mqtt: [https://www.zigbee2mqtt.io/guide/configuration/device-availability.html#availability-advanced-configuration](url)

aggiungendo al docker-campose.yaml la stringa `availability: true` sotto la sezione di z2m e riavviando il servizio, i miei dispositivi sono on da ieri sera. Vediamo se riescono a tenere botta.

**Reply by snakuzzo** [reply]
Non vedo alcun legame con quello che era il tuo problema. Questa non è un'impostazione "necessaria"

**Reply by calcioscacchi** [reply]
infatti sono tornati a cadere. Comunque, questo è uno degli output dell'ascolto che mi hai suggerito, anche se ho mess ocome ascolto "#".

`{
    "availability": [
        {
            "topic": "zigbee2mqtt/bridge/state"
        }
    ],
    "device": {
        "identifiers": [
            "zigbee2mqtt_0x00158d000805df6c"
        ],
        "manufacturer": "Xiaomi",
        "model": "Aqara human body movement and illuminance sensor (RTCGQ11LM)",
        "name": "Aquara movimento cucina",
        "sw_version": "3000-0001"
    },
    "device_class": "motion",
    "json_attributes_topic": "zigbee2mqtt/Aquara movimento cucina",
    "name": "Aquara movimento cucina occupancy",
    "payload_off": false,
    "payload_on": true,
    "state_topic": "zigbee2mqtt/Aquara movimento cucina",
    "unique_id": "0x00158d000805df6c_occupancy_zigbee2mqtt",
    "value_template": "{{ value_json.occupancy }}"
}`

i dispositivi sono andati in "non disponibile" dalle 5:45 di questa mattina.
![image](https://user-images.githubusercontent.com/60674280/186017513-b731a631-1e40-4ec1-95cf-4f29a3b3bf18.png)


se riavvio zigbee2mqtt, tornano su.
![image](https://user-images.githubusercontent.com/60674280/186017942-b689a1e4-e848-47e5-8c99-410e05fafdd5.png)

**Comment by calcioscacchi** [comment]
Ho visto che il mio problema era una cosa nota, dovrebbe essere stato risolto con l'ultimo aggiornamento di Z2M

**Reply by snakuzzo** [reply]
Sì, visto...così pare.
